js/index.js

```diff
@@ -68,10 +76,25 @@ const pageTimeout = setTimeout(() => {
     } else {
       errorMessage.textContent = '';
       errorMessage.style.display = 'none';
+      localStorage.setItem('contact-info', JSON.stringify(contactInfo));
+    }
+  });
+  const setContactValues = () => {
+    if (Object.keys(contactInfo).length) {
+      contactForm.name.value = contactInfo.name;
+      contactForm.email.value = contactInfo.email;
+      contactForm.message.value = contactInfo.message;
```

Comment:
[OPTIONAL] Well done on persisting the data 📊 but when I refresh the page the message isn't returned, at least not the one I wrote. The reason being your message is only persisted once I leave the input. See below 👇🏽 
![Screenshot 2023-02-02 at 08 28 05](https://user-images.githubusercontent.com/45739963/216337689-3a0f1643-e318-486b-ac12-0e0401c3b098.png)

You are trying to access a key message but this key doesn't exist so it returns undefined. 🤒 
![Screenshot 2023-02-02 at 08 28 17](https://user-images.githubusercontent.com/45739963/216337745-2905ed35-d1e8-4201-bdad-3c684b3e5231.png)

I'm still approving because you did great work and the project is working 🎉  but consider fixing this as this can be an issue if the page refreshes before the user leaves it will return undefined. 😿  I marked it as [OPTIONAL] but addressing this would make your project better. Keep going, you rock. 💪🏽

Reply:
Thank you for pointing out the issues. I will work on that🙏👌

Reply:
@didierganthier Hello,
Thank you for pointing out those minor issue. I will work on that🙏👌